src/components/docs/versions/builds/builds.module.scss

```diff
@@ -12,4 +12,14 @@
 }
 
 .expandedContentRow {
+  * {
+    white-space: pre-wrap;
+  }
```

Comment:
Not sure if we really need a wildcard here. The page feels a bit heavy, might be because of this.

Reply:
The page used to not work without it, but now it seems that it does.